### PR TITLE
docs: Added Flightcontrol Deployment Option

### PR DESCRIPTION
**Pull request by ModupeD**
Flightcontrol fully automates your deployments to Amazon Web Services (AWS)

### CODE REVIEW

**Review by flybayer — APPROVED** [review]

**Comment by michaelessiet** [comment]
Great, PR. Please can you add a few images to the instructions under the Dashboard section to make it a bit easier for visual learners to follow

**Comment by ModupeD** [comment]
@michaelessiet I have added the images

**Comment by michaelessiet** [comment]
> @michaelessiet I have added the images

Thanks Modupe, I'll make sure to merge it in this weekend

File: content/guides/how-to-guides/deployment/deploying-to-flightcontrol.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import { Aside } from "~/components/configurable/Aside";
+
+<Title>Deploying to AWS via Flightcontrol</Title>
+
+Here we're going to use [Flightcontrol](https://www.flightcontrol.dev/?ref=solid) to deploy our Solid project.
```
Comment (CBID2) [inline content/guides/how-to-guides/deployment/deploying-to-flightcontrol.mdx:5]: [Typo]
```suggestion
Here, we're going to use [Flightcontrol](https://www.flightcontrol.dev/?ref=solid) to deploy our Solid project.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ModupeD! :) This is off to a great start! :) I just pointed out some typos and other things to consider revising! :)

**Pull request merged**